core/src/main/java/com/linecorp/armeria/client/AbstractClientFactory.java

```diff
@@ -42,12 +42,75 @@
         return newClient(URI.create(uri), clientType, options);
     }
 
+    /**
+     * Creates a new client that connects to the specified {@link URI} using the default
+     * {@link ClientFactory}.
+     *
+     * @param uri the URI of the server endpoint
+     * @param clientType the type of the new client
+     * @param options the {@link ClientOptionValue}s
+     */
     @Override
     public final <T> T newClient(URI uri, Class<T> clientType, ClientOptionValue<?>... options) {
         requireNonNull(options, "options");
         return newClient(uri, clientType, ClientOptions.of(options));
     }
 
+    /**
+     * Creates a new client that connects to the specified {@link Endpoint} with the {@link Scheme} using
+     * the default {@link ClientFactory}.
```

Comment:
This Javadoc block is unnecessary and incorrect. Note that a method description is inherited from the method it overrides/implements.